pkg/relayer/session/session.go

```diff
@@ -261,3 +265,9 @@ func (rs *relayerSessionsManager) mapAddMinedRelayToSessionTree(
 	// Skip because this map function only outputs errors.
 	return nil, true
 }
+
+// checkGracePeriodEnd checks if the grace period for the session has ended
```

Comment:
Since it just returns a bool, I think something like `isWithinGracePeriod` or `isOutsideGracePeriod` or `didGracePeriodEnd`, etc is more appropriate based on how you're thinking about it.